packages/react-scripts/template/README.md

```diff
@@ -936,6 +937,53 @@ Install the Surge CLI if you haven't already by running `npm install -g surge`.
 
 Note that in order to support routers that use html5 `pushState` API, you may want to rename the `index.html` in your build folder to `200.html` before deploying to Surge. This [ensures that every URL falls back to that file](https://surge.sh/help/adding-a-200-page-for-client-side-routing).
 
+## Forking as Alternative to Ejecting
+You can fork `react-scripts`, publish your fork, and then use it with Create React App.
+
+Pros:
+- You can customize your setup (eg: add CSS Modules, PostCSS plugins, decorators, etc…)
+- You still have all Create React App features and updates
+
+Cons:
+- You need to maintain your fork, [make sure it is synced](https://help.github.com/articles/fork-a-repo/#keep-your-fork-synced) with the upstream to have all updates. [Backstroke](https://github.com/1egoman/backstroke) is a bot that can help you with this.
+
+First, fork repository [`create-react-app`](https://github.com/facebookincubator/create-react-app) repository.
+
+Next, change the name of `react-scripts` package to the name you chose for your fork, and also reset its version. It is recommended to use [scoped package](https://docs.npmjs.com/misc/scope), ie: `@yourcompany/react-scripts`.
+### /packages/react-scripts/package.json
+```js
+{
+  "name": "@yourcompany/react-scripts",
+  "version": "0.0.1",
+  …
+}
+```
+
+Now make your changes inside `react-scripts` package. For example, add CSS Modules:
```

Comment:
I  tested and renaming `react-scripts` as suggested in the instructions breaks `npm start` because it cannot find `react-scripts/scripts/start.js` and `react-scripts/tasks/cra.sh`.

Should we not rename `react-scripts` package in the fork? Just rename the fork? If so we need to clarify `Next, change the name of `react-scripts` package to the name you chose for your fork`.